### PR TITLE
Map can be zommed after clicking on it.

### LINKED ISSUE #54: Only make map zoomable after initial click on area
Currently when a user scrolls down with the mouse and the cursor is suddenly over the map the map zooms in instead of the page scrolling down further.

Expected: Only zoom map once the use has clicked on the map area.

### LINKED ISSUE #245: '+' and '-' button of the map doesn't work
bug : 
currently the '+' and '-' button on the map doesn't work. 

suggested improvement:
The map should zoom in and out when the '+' or '-' button is clicked.

I want to work on this issue.

### PR DESCRIPTION
**Pull request by nityanandagohain**
Now when we want to zoom in and out on the map clicking on the map activate the zooming functionality and the zoom buttons are also activated.

fixes #245 
fixes #54 
preview https://nityanandagohain.github.io/2018.fossasianitya.io/

### CODE REVIEW

**Comment by nityanandagohain** [comment]
@simsausaurabh  but I think @mariobehling wanted it after clicking on it #54  and @duttaditya18  also suggested like that in #245 

regarding the second issue I am figuring it out .

**Comment by nityanandagohain** [comment]
I have updated it and squashed the commits

File: index.html [hunk 1]
```diff
@@ -3110,6 +3110,12 @@ <h1 class="large-h1">Subscribe to stay informed</h1>
         });
 
         });
-      </script>
+        </script>
```
Comment (realslimshanky) [inline index.html]: No need to separate out this script section from the one below it.

Comment (realslimshanky) [inline index.html]: No need to separate out this script section from the one below it. Please remove this closing script tag.

File: index.html [hunk 2]
```diff
@@ -3110,6 +3110,12 @@ <h1 class="large-h1">Subscribe to stay informed</h1>
...
+
+        <script>
```
Comment (realslimshanky) [inline index.html]: Please remove this opening script tag and merge the two sections together.

Comment (nityanandagohain) [reply index.html]: ok

**Pull request merged**